packages/evm/src/fungibleAssetTransfer.ts

```diff
@@ -130,6 +145,10 @@ class FungibleAssetTransfer extends AssetTransfer {
   public async getApprovalTransactions(
     overrides?: ethers.Overrides,
   ): Promise<Array<TransactionRequest>> {
+    if (this.isNativeTransfer()) {
+      return [];
```

Comment:
if I'm reading this correctly, if we are in the scope of native transfer, there is no need to get approvals, hence we return empty array right?  If this is the case, does it make sense to include this logic -native transfer- into a method o function call that makes you add this branching here? would it make sense to avoid returning the empty array? 🤔

Reply:
already discussed and solved